packages/pwa-kit-dev/src/configs/webpack/overrides-plugin.js

```diff
@@ -0,0 +1,144 @@
+/*
```

Comment:
This file is the majority of the logic, though a bunch is done in `pwa-kit-dev/src/configs/wepack/config.js`